### PR TITLE
feat: user profile update data

### PR DESCRIPTION
**Pull request by aguirrekarina**
# What did I do?

Improve user profile update flow

- Implement user profile updates for name, email, and password
- Add email verification requirement.
- Validate current password for sensitive updates

## Code reviewers
- @denis-gandel 
- @JeferssonCL

### CODE REVIEW

**Comment by denis-gandel** [comment]
Screenshot from 2024-11-18 11-35-29

Try to give it a better style, because when I tried to change the information I didn't realize that I should press and change to the field before hitting the “Update Profile” button.

The fields are not very noticeable or intuitive to change the data, you could use the Inputs created for the Create Store.

**Comment by denis-gandel** [comment]
> Screenshot from 2024-11-18 11-35-29
> 
> Try to give it a better style, because when I tried to change the information I didn't realize that I should press and change to the field before hitting the “Update Profile” button.
> 
> The fields are not very noticeable or intuitive to change the data, you could use the Inputs created for the Create Store.

In addition, the type of account is unnecessary information.

File: apps/merchant-web/src/components/profile-management/ProfileForm.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
```
Comment (denis-gandel) [inline apps/merchant-web/src/components/profile-management/ProfileForm.tsx:1]: Try not to import React, only import what is necessary.

**Comment by denis-gandel** [comment]
Do not forget to follow the guideline defined for the frontend.

**Review by denis-gandel — APPROVED** [review]
Good job :D

**Review by JeferssonCL — APPROVED** [review]
Good Job

**Pull request merged**